terrain-tiles/README.md

```diff
@@ -0,0 +1,48 @@
+
+# Terrain Tiles
+
+Contains terrain tiles around Innsbruck.
```

Comment:
Please provide the center in lat/lon or even better a code example of how to use/see these tiles.

Reply:
This tile is for the "N047E011" region. This was picked to match the debug/terrain.html center location [11.40416, 47.26475], though that is not quite the center of the terrainrgb tile.

I think, once approved, this would be used like this
```
terrain: {
	type: 'raster-dem',
	url: 'https://demotiles.maplibre.org/terrain-tiles/tiles.json',
	tileSize: 256
},
```
However, that won't work until this is approved. Right now my test page at 
https://stackblitz.com/edit/web-platform-x1xjty?file=index.html
is pointing directly to github like this

terrain: {
	type: 'raster-dem',
	url: 'https://raw.githubusercontent.com/acalcutt/demotiles/example_tiles/terrain-tiles/tiles.json',
	tileSize: 256
},

Reply:
I know, but the readme won't be added until this is approved too, so there's no problem.
All I'm asking here is to improve the readme documentation so that if anyone finds this folder and would like to use these tiles it would be easy for him.
Thanks!!

Reply:
If I wanted to make a index page like https://github.com/maplibre/demotiles/blob/gh-pages/index.html , is there a way to link to the latest terrain3d javascript file and css right now?

Reply:
Yes, we have it in the gh pages of maplibre which is reflected to maplibre.org, under pull folder, but I wouldn't recommend as it is temporary...

Reply:
I made an attempt at the readme fixes here https://github.com/wipfli/demotiles/pull/3 if @wipfli approves that